understanding/20/contrast-enhanced.html

```diff
@@ -136,11 +153,22 @@ <h3>Rationale for the Ratios Chosen</h3>
             [[ARDITI]].
          </p>
          
+         <div class="note">
+            
+            <p>Some people with cognitive disabilities require color combinations or hues that have
+               low contrast, and therefore we allow and encourage authors to provide mechanisms to
+               adjust the foreground and background colors of the content. Some of the combinations
+               that could be chosen may have contrast levels that will be lower than those found
+               in the Success Criteria. This is not a violation of this Success Criterion, provided
+               there is a mechanism that will return to the default values set out here.
```

Comment:
```suggestion
               there is a mechanism that will return to the required values set out here.
```

Reply:
sure, makes sense. tweaked it a bit further, for better readability.